spec/Section 7 -- Response.md

```diff
@@ -244,6 +264,161 @@ discouraged.
 }
 ```
 
+### Incremental
+
+The `incremental` entry in the response is a non-empty list of Defer or Stream
+payloads. If the response of the GraphQL operation is a response stream, this
+field may appear on both the initial and subsequent values.
```

Comment:
"this field" - maybe "this entry" since "field" has specific meaning in GraphQL.